tests/dummy/app/templates/index-example.hbs

```diff
@@ -0,0 +1,53 @@
+<section>
+  <h1>Create a new user</h1>
+  <form
+    id="userForm"
+    {{verify-form-validity submit=this.handleFormSubmit}}
+    {{on "validated" this.assignValidationMessages}}
+  >
+    <div class="full-width text-right"><sup class="required">*</sup> Required</div>
+
+    <label for="userForm-name">Name<sup class="required">*</sup></label>
+    <input id="userForm-name" name="name" required {{validity}}>
+    <span class="validation-message">{{this.validationMessages.name}}</span>
+
+    <label for="userForm-phone">Phone</label>
+    <input id="userForm-phone" name="phone" type="tel" {{validity}}>
+    <span class="validation-message">{{this.validationMessages.phone}}</span>
+
+    <label for="userForm-email">Email<sup class="required">*</sup></label>
+    <input id="userForm-email" name="email" type="email" required {{validity}}>
+    <span class="validation-message">{{this.validationMessages.email}}</span>
+
```

Comment:
Add a select for demo purposes? (idk, something like "Preferred contact method" to stick with the demo theme?)